libuavcan/include/uavcan/transport/can_acceptance_filter_configurator.hpp

```diff
@@ -34,23 +36,33 @@ namespace uavcan
  */
 class CanAcceptanceFilterConfigurator
 {
+public:
+	/**
+	 * These arguments defines whether acceptance filter configuration has anonymous messages or not
+	 */
+	enum AnonMessagePresence {WithAnonMsg,NoAnonMsg};
```

Comment:
I'm a bit paranoid about naming, so let me be an annoyance. Can we name this stuff like `AcceptAnonymousMessages`, `IgnoreAnonymousMessages`?